modules/custom/openy_activity_finder/js/programs_search.js

```diff
@@ -314,7 +314,6 @@
         var query = [];
         query.push('log=' + encodeURIComponent(this.table[index].log_id));
         query.push('details=' + encodeURIComponent(this.table[index].name));
-        query.push('nid=' + encodeURIComponent(this.table[index].nid));
```

Comment:
why do we remove nid in the query?

Reply:
Whoops must have been a mistake. Added back in.